src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java

```diff
@@ -31,17 +34,20 @@
 import javax.validation.Valid;
 
 import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
 import static uk.gov.hmcts.reform.sscs.ccd.domain.DwpState.HEARING_DATE_ISSUED;
 import static uk.gov.hmcts.reform.sscs.helper.service.CaseHearingLocationHelper.mapVenueDetailsToVenue;
 import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.LISTED;
 
 @Slf4j
 @Service
 @RequiredArgsConstructor
+@SuppressWarnings("PMD.ExcessiveImports")
```

Comment:
Could .domain.* here to avoid this suppression.

Reply:
When I do that it complains that it is not in use